# msg to pick up marker
String aruco_name # name of aruco frame assoc. w/ desired marker
